src/topic/components/__tests__/TopicsList.test.js

```diff
@@ -41,4 +46,14 @@ describe('<TopicsList>', () => {
         });
     });
     
+    describe('when topics are loading', () => {
+        beforeEach(() => {
+            wrapper = shallow(<TopicsList pending={true}/>);
```

Comment:
`const` missing

Reply:
I don't think it is necessary. `let wrapper` is defined at the very beginning of the test. Or you mean sosmething else than `const wrapper` ? (btw it wouldn't be visible in the test later or, so rather `let wrapper` inside describe but this is already done above)

Reply:
eh, haven't noticed /shrug leave it